.pipelines/singletenancy/dualstack-overlay/dualstackoverlay-e2e-step-template.yaml

```diff
@@ -75,4 +75,41 @@ steps:
       echo "Dualstack Overlay Linux datapath IPv4 test"
       sudo go test -count=1 datapath_linux_test.go -timeout 1m -tags connection -run ^TestDatapathLinux$ -tags=connection,integration
     name: "DualStack_Overlay_Linux_Tests"
-    displayName: "DualStack Overlay Linux Tests"
\ No newline at end of file
+    displayName: "DualStack Overlay Linux Tests"
+
+  - task: AzureCLI@2
+    inputs:
+      azureSubscription: $(AZURE_TEST_AGENT_SERVICE_CONNECTION)
+      scriptLocation: "inlineScript"
+      scriptType: "bash"
+      addSpnToEnvironment: true
+      inlineScript: |
+        make -C ./hack/aks windows-nodepool-up AZCLI=az VM_SIZE=$(VM_SIZE) NODE_COUNT=$(NODE_COUNT) SUB=$(SUB_AZURE_NETWORK_AGENT_TEST) CLUSTER=${{ parameters.clusterName }}-$(make revision)
+        echo "Windows node are successfully added to Dualstack Overlay Cluster"
+        kubectl cluster-info
+        kubectl get node
+        kubectl get po -owide -A
+        echo "Deploying windows CNS"
+        kubectl apply -f test/integration/manifests/cns/daemonset-windows.yaml
+        kubectl get po -owide -A
+        echo "Waiting for CNS pods to be running"
+        sleep 45
+        echo "Upload CNI"
+        dropgz_version=$(make cni-dropgz-version)
+        export DROP_GZ_URL=$( make cni-dropgz-test-image-name-and-tag OS='windows' ARCH='amd64' OS_VERSION='ltsc2022' CNI_DROPGZ_VERSION=${dropgz_version})
+        envsubst < ./test/integration/manifests/cni/cni-installer-v1-windows.yaml | kubectl apply -f -
+    name: "Add_Windows_Node"
+    displayName: "Add windows node on DualStackOverlay cluster"
+
+  - script: |
+      cd test/integration/load
+      sudo go test -timeout 30m -tags load -run ^TestLoad$ -tags=load -os=windows
+      echo "DualStack Overlay Windows control plane CNS validation test"
+      sudo go test -timeout 30m -tags load -cni dualstack -run ^TestDualStackProperties$ -tags=load -os=windows 
+      echo "DualStack Overlay Windows control plane Node properties test"
+      sudo go test -timeout 30m -tags load -cni dualstack -run ^TestValidateState$ -tags=load -os=windows
+      echo "DualStack Overlay Windows datapath test"
+      cd ../datapath
+      sudo go test -count=1 datapath_windows_test.go -timeout 3m -tags connection -run ^TestDatapathWin$ -tags=connection
```

Comment:
Might be good to split these 4 tests into different pipeline steps so the log output is more clear when one test fails